qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -5,7 +5,7 @@
 #    guest OS really supported it.
 - hotplug_memory:
     type = hotplug_mem
-    mem_fixed = 1024
+    mem_fixed = 4096
```

Comment:
how to deal with guest (such as linux) that don't use 4096 memory ? and also host free memory is less than 4096 when booting with other windows guest or linux guest?

Reply:
@jingzhao84 It's ok to boot guest with 4G memory even if its minimum mem is less than that. If host free memory is less than 4G(not much), linux guest would be fine to boot up.  Other windows guest might not only when host system couldn't manage to squeeze more memory for it, such as using swap.  I'd say the scenario is not common.  If the host memory is very small,  we probably should not use the host to test memory cases. Thanks!